src/styles/blocks/process.scss

```diff
@@ -0,0 +1,316 @@
+    }
+
+    &--yellow {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $yellow-color;
+      background-color: $dark-yellow-color;
+    }
+
+    &--green {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $green-color;
+      background-color: $dark-green-color;
+    }
+
+    &--pink {
+      @extend %poppinsIconBlue;
+      width: 46px;
+      height: 24px;
+      border-radius: 23px;
+      text-align: center;
+      text-transform: uppercase;
+      color: $pink-color;
+      background-color: $dark-pink-color;
+    }
+  }
+
+  &__text {
+    @extend %openSansContent13;
+    color: $gray-color;
+    padding: 24px 0 32px;
+  }
+
+  &__learn-more {
+    @extend %poppinsButtonText;
+    line-height: 40px;
+    display: block;
+    background-image: url(../images/services/Arrow.svg);
+    background-size: 40px;
+    background-repeat: no-repeat;
+    text-decoration: none;
+    color: $gray-blue-color;
+
+    &::after {
+      content: "Learn more";
+      position: relative;
+      left: 56px;
+    }
+
+    &--blue {
+      @extend %poppinsButtonText;
+      line-height: 40px;
+      display: block;
+      background-image: url(../images/services/Arrow-blue.svg);
+      background-size: 40px;
+      background-repeat: no-repeat;
+      text-decoration: none;
+      color: $gray-blue-color;
+
+      &::after {
+        content: "Learn more";
+        position: relative;
+        left: 56px;
+      }
+    }
+
+    &--yellow {
+      @extend %poppinsButtonText;
+      line-height: 40px;
+      display: block;
+      background-image: url(../images/services/Arrow-yellow.svg);
+      background-size: 40px;
+      background-repeat: no-repeat;
+      text-decoration: none;
+      color: $gray-blue-color;
+
+      &::after {
+        content: "Learn more";
+        position: relative;
+        left: 56px;
+      }
+    }
+
+    &--green {
+      @extend %poppinsButtonText;
+      line-height: 40px;
+      display: block;
+      background-image: url(../images/services/Arrow-green.svg);
+      background-size: 40px;
+      background-repeat: no-repeat;
+      text-decoration: none;
+      color: $gray-blue-color;
+
+      &::after {
+        content: "Learn more";
+        position: relative;
+        left: 56px;
+      }
+
+    }
+
+    &--pink {
+      @extend %poppinsButtonText;
+      line-height: 40px;
+      display: block;
+      background-image: url(../images/services/Arrow-pink.svg);
+      background-size: 40px;
+      background-repeat: no-repeat;
+      text-decoration: none;
+      color: $gray-blue-color;
+
+      &::after {
+        content: "Learn more";
+        position: relative;
+        left: 56px;
+      }
+    }
+
+    &:hover {
+      transform: scale(1.1) translateY(-50%);
```

Comment:
Also this transition is a bit odd. It is not smooth and works when hovering not the button but its background. Please fix this